app/aggregates/reviewing/available_reviewer_actions.rb

```diff
@@ -0,0 +1,56 @@
+module Reviewing
+  class AvailableReviewerActions
+    AVAILABLE_ACTIONS = {
```

Comment:
Are cifc apps accounted for?

Reply:
It behaves like an Means application, but I'll add specs to make that clear.